bindings/python/src/utils/mod.rs

```diff
@@ -22,7 +22,7 @@ pub fn extract_regions_from_py_any(regions: &Bound<'_, PyAny>) -> Result<RegionS
 
         let regions = gtars::common::models::RegionSet::try_from(regions);
         match regions {
-            Ok(regions) => return Ok(RegionSet::from(regions)),
+            Ok(regions) => return Ok(regions),
```

Comment:
Is it correct change?. It seems it does nothing

Reply:
True -- I agree I dont know let me look into this

Reply:
Yeah I can break early using the `?` operator:
```rust
let regions = gtars::common::models::RegionSet::try_from(regions)?;
return Ok(regions);
```